glide.yaml

```diff
@@ -15,4 +15,10 @@ import:
 - package: github.com/Azure/go-autorest
   version: 10.14.0
 - package: github.com/sirupsen/logrus
-  version: v1.0.6
\ No newline at end of file
+  version: v1.0.6
+- package: github.com/ghodss/yaml
+  version: e9ed3c6dfb39bb1a32197cb10d527906fe4da4b6  # Master as per 2018-08-13
+- package: github.com/go-test/deep
```

Comment:
Just rambling but I think we can get rid of this package by using `k8s.io/apimachinery/pkg/equality.Semantic.DeepEqual` and `k8s.io/apimachinery/pkg/util/diff.ObjectGoPrintDiff`.